buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt

```diff
@@ -110,12 +113,12 @@ object Dependencies {
     const val exoMedia = "com.github.brianwernick:ExoMedia:$EXOMEDIA_VERSION"
     const val firebaseAnalytics = "com.google.firebase:firebase-analytics-ktx:$FIREBASE_ANALYTICS_VERSION"
     const val firebaseCrashlytics = "com.google.firebase:firebase-crashlytics:$FIREBASE_CRASHLYTICS_VERSION"
+    const val firebaseCrashlyticsPlugin =
```

Comment:
We are using few firebase dependencies, but what do you think about using BoM for it?

Reply:
See https://github.com/GetStream/stream-chat-android/pull/784